touchforms/backend/touchcare.py

```diff
@@ -248,25 +256,32 @@ def getIDsForValue(self, field_name, value):
 
 class CCInstances(InstanceInitializationFactory):
 
-    def __init__(self, sessionvars, api_auth):
+    def __init__(self, sessionvars, api_auth, form_context):
         self.vars = sessionvars
         self.auth = api_auth
         self.fixtures = {}
+        self.form_context = form_context or {}
 
     def generateRoot(self, instance):
         ref = instance.getReference()
-    
         def from_bundle(inst):
             root = inst.getRoot()
             root.setParent(instance.getBase())
             return root
 
         if 'casedb' in ref:
-            return CaseInstanceTreeElement(instance.getBase(), 
-                        CaseDatabase(self.vars.get('host'), self.vars['domain'], self.vars['user_id'], 
-                                     self.auth, self.vars.get("additional_filters", {}),
-                                     self.vars.get("preload_cases", False)),
-                        False)
+            return CaseInstanceTreeElement(
+                instance.getBase(),
+                CaseDatabase(
+                    self.vars.get('host'),
+                    self.vars['domain'],
+                    self.auth,
```

Comment:
oh sorry no i removed it on purpose. CaseDatabase was subclassing TouchfromsStorageUtility and needlessly adding the parameter: https://github.com/dimagi/touchforms/pull/118/files#diff-55395e6e3dc3202a4b20e5f392679013L170